File: lec5/struct0.c
#include <stdio.h>
#include <cs50.h>
#include <ctype.h>

#include "struct.h"

int main (void){

    //allocate space for students
    int enrollment = get_int("Enrollment: ");
    //struct named students of length enrollment
    student students[enrollment];

    //prompt for student names and dorm
    for(int i = 0; i < enrollment ; i++){

        students[i].name = get_string("name: ");
        students[i].dorm = get_string("dorm: ");

    }

    //print it all out

    for(int i = 0; i < enrollment; i++){

        printf("Student name is %s and stays in %s\n", students[i].name, students[i].dorm);
    }

    // problem is that it does not save the data, throws it away



}

File: lec8/struct0.py
from cs50 import get_string
from student import Student

students = []

for i in range(3):
    name = get_string("name: ")
    dorm = get_string("dorm: ")
    s = Student(name, dorm)
    students.append(s)

for student in students:
    print(f"{student.name} stays in {student.dorm}")